source/content/posts/Formulario_de_Registro.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Formulario de Registro"
+date: 2018-01-30T16:31:49Z
+draft: false
+---
+
+# ¿Cómo crear un formulario de registro utilizando Scroll View?
```

Comment:
I think that you might change the phrase "Scroll View" to "el componente Scroll View"